version

```diff
@@ -1 +1 @@
-0.9.5
\ No newline at end of file
+0.9.6-rc.2
```

Comment:
Not sure we should be sending out a release candidate to TestNet? Should we not release it as 0.9.6 (and 0.9.7 for decimal precision?).